multiaccounts/accounts/database.go

```diff
@@ -937,6 +940,6 @@ func (db *Database) GetAccountLastSynced(address types.Address) (uint64, error)
 func (db *Database) SetAccountLastSynced(address types.Address, clock uint64) error {
 	query := "UPDATE accounts SET clock = ? WHERE address = ? AND clock < ?"
 
-	_, err := db.db.Exec(query, address, clock)
+	_, err := db.db.Exec(query, clock, address, clock)
```

Comment:
Why passing clock twice?

Reply:
`query := "UPDATE accounts SET clock = ? WHERE address = ? AND clock < ?"`

The query has 3 positional parameters marked with `?`, we need to give a value for each of the parameters in the query.

`SET clock = ?` requires the incoming clock value
`WHERE address = ?` requires the address
`clock < ?` requires the clock again